code/modules/vehicles/mecha/equipment/weapons/greyscale_weapons.dm

```diff
@@ -0,0 +1,385 @@
+		MECHA_L_ARM = list("N" = list(-4,30), "S" = list(32,12), "E" = list(52,34), "W" = list(-22,14)),
+	)
+	ammotype = /datum/ammo/bullet/smg/mech
+	projectiles = 60
+	projectiles_cache = 420
+	projectiles_cache_max = 420
+	variance = 20
+	projectile_delay = 0.15 SECONDS
+	slowdown = 0.15
+	harmful = TRUE
+	ammo_type = MECHA_AMMO_SMG
+	fire_mode = GUN_FIREMODE_AUTOMATIC
+
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/burstrifle
+	name = "\improper Tethys burst rifle"
+	icon = 'icons/mecha/mecha_equipment_64x32.dmi'
+	desc = "Medium-sized burstfire weapon. Similar to the Cronus assault rifle, but burstfires."
+	icon_state = "burstrifle"
+	fire_sound = 'sound/mecha/weapons/mech_rifle.ogg'
+	mech_flags = EXOSUIT_MODULE_GREYSCALE
+	flash_offsets = list(
+		MECHA_R_ARM = list("N" = list(36,30), "S" = list(-2,-6), "E" = list(64,17), "W" = list(-33,37)),
+		MECHA_L_ARM = list("N" = list(-4,30), "S" = list(32,-6), "E" = list(63,37), "W" = list(-34,17)),
+	)
+	ammotype = /datum/ammo/bullet/rifle/mech/burst
+	projectiles = 72
+	projectiles_cache = 360
+	projectiles_cache_max = 360
+	variance = 15
+	projectile_delay = 0.6 SECONDS
+	burst_amount = 3
+	projectile_burst_delay = 0.1 SECONDS
+	slowdown = 0.25
+	harmful = TRUE
+	ammo_type = MECHA_AMMO_BURSTRIFLE
+	fire_mode = GUN_FIREMODE_AUTOBURST
+
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/assault_rifle
+	name = "\improper Cronus assault rifle"
+	icon = 'icons/mecha/mecha_equipment_64x32.dmi'
+	desc = "The stock-and-standard extra-sized multipurpose rifle for TGMC mech units."
+	icon_state = "assaultrifle"
+	fire_sound = 'sound/mecha/weapons/mech_rifle.ogg'
+	mech_flags = EXOSUIT_MODULE_GREYSCALE
+	flash_offsets = list(
+		MECHA_R_ARM = list("N" = list(36,30), "S" = list(-2,-6), "E" = list(64,17), "W" = list(-34,37)),
+		MECHA_L_ARM = list("N" = list(-4,30), "S" = list(32,-6), "E" = list(64,37), "W" = list(-34,17)),
+	)
+	ammotype = /datum/ammo/bullet/rifle/mech
+	projectiles = 50
+	projectiles_cache = 300
+	projectiles_cache_max = 300
+	variance = 15
+	projectile_delay = 0.2 SECONDS
+	slowdown = 0.2
+	harmful = TRUE
+	ammo_type = MECHA_AMMO_RIFLE
+	fire_mode = GUN_FIREMODE_AUTOMATIC
+
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/shotgun
+	name = "\improper Phoebe shotgun"
+	icon = 'icons/mecha/mecha_equipment_64x32.dmi'
+	desc = "The TGMC classic weapon, but bigger and better! Fires plus-sized buckshot for high damage in close combat."
+	icon_state = "shotgun"
+	fire_sound = 'sound/mecha/weapons/mech_shotgun.ogg'
+	mech_flags = EXOSUIT_MODULE_GREYSCALE
+	flash_offsets = list(
+		MECHA_R_ARM = list("N" = list(36,30), "S" = list(-2,-4), "E" = list(61,16), "W" = list(-31,36)),
+		MECHA_L_ARM = list("N" = list(-4,30), "S" = list(32,-4), "E" = list(61,36), "W" = list(-31,16)),
+	)
+	ammotype = /datum/ammo/bullet/shotgun/mech
+	projectiles = 10
+	projectiles_cache = 50
+	projectiles_cache_max = 50
+	variance = 6
+	projectile_delay = 2
```

Comment:
the projectile_delay of 2 is probably translating to 0.2 because the shotgun is firing as fast as an assault rifle, should be 2.0 probably